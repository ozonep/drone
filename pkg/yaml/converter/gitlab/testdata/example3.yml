image:
  name: ruby:2.2
  entrypoint: ["/bin/bash"]

services:
- name: my-postgres:9.4
  alias: db-postgres
  entrypoint: ["/usr/local/bin/db-postgres"]
  command: ["start"]

before_script:
- bundle install

test:
  script:
  - bundle exec rake spec