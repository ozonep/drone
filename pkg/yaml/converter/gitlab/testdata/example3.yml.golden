---
kind: pipeline
name: test

platform:
  os: linux
  arch: amd64

steps:
- name: test
  image: ruby:2.2
  commands:
  - bundle install
  - bundle exec rake spec

services:
- name: db-postgres
  image: my-postgres:9.4
  entrypoint:
  - /usr/local/bin/db-postgres
  command:
  - start

...
